frontend/packages/console-shared/src/components/toast/ToastProvider.tsx

```diff
@@ -58,7 +58,7 @@ const ToastProvider: React.FC = ({ children }) => {
               variant={toast.variant}
               timeout={toast.timeout}
               onTimeout={() => removeToast(toast.id)}
-              data-test={`${toast.title} alert`}
+              data-test={toast.dataTest || 'alert'}
```

Comment:
I need to run the full e2e tests to check if this change impact or break any existing test(s).

Reply:
```suggestion
              data-test={toast.dataTest || `${toast.title} alert`}
```